carbon-location.html

```diff
@@ -28,6 +28,13 @@
       path: '/elements/carbon-route-converter'
     }
 
+The following queryParams object will also be produced:
```

Comment:
`carbon-location` doesn't expose anything named `queryParams` so this could be misleading. I'd recommend leaving that off.